api/services/deckService.js

```diff
@@ -0,0 +1,26 @@
+
+export default class Deck{
+    constructor(whiteCardDeck, players) {
+        this.whiteCardDeck = whiteCardDeck.map(cards => JSON.parse(JSON.stringify(cards)))
```

Comment:
why is it calling JSON.parse and JSON.stringify - dont they cancel each other out?

Reply:
If I run just JSON.parse it fails. Adding JSON.stringify makes it work :(

Reply:
but can't you just run neither?

Reply:
I mean, it passes the tests, but then i don't really understand this:  this.whiteCardDeck = whiteCardDeck.map(cards => (cards))

Reply:
if thats the case its probably not nbecessary at all! just replace with `this.whiteCardDeck = whiteCardDeck`

Reply:
That broke everything :'(